test/test.js

```diff
@@ -194,6 +194,15 @@ describe('Datastore', () => {
       wmsUrl);
   });
 
+  it('can create a WMTS Store', async () => {
+    // TODO: make sure the WMS actually exists
```

Comment:
Copy/paste? can this be removed?

Reply:
This note shall be a reminder, that the provided GeoServer might be down and therefore the tests fail. But we can also convert this to an issue.

Reply:
true, I fixed it with e232d88